### PR TITLE
feat: get hover color

### PR DESCRIPTION
**Pull request by cbt1**
**White and transparent background color**

https://github.com/qlik-oss/nebula-table-utils/assets/16608020/6b0560d0-002e-4795-91f8-1670011b038d

**Dark background color**

https://github.com/qlik-oss/nebula-table-utils/assets/16608020/0fc456eb-731d-43cc-a0ef-fa11dc37c71d

**Bright background color**

https://github.com/qlik-oss/nebula-table-utils/assets/16608020/d22776ae-cfb6-4e74-b9d4-8ea07b102ea0

### CODE REVIEW

File: jest.config.cjs [hunk 1]
```diff
@@ -5,12 +5,13 @@ module.exports = {
...
   transform: {
+    '\\.js$': 'babel-jest',
     '^.+\\.(t|j)sx?$': '@swc/jest',
   },
   moduleNameMapper: {
     '@qlik-trial/sprout/icons/.*': '<rootDir>/test-utils/sprout-icons-mock.tsx',
   },
   coveragePathIgnorePatterns: ['dist', 'lib', 'build-utils', './src/constants'],
-  transformIgnorePatterns: ['<rootDir>/node_modules/(?!(cliui|wrap-ansi)/*)'],
+  transformIgnorePatterns: ['node_modules/.pnpm/(?!(cliui|wrap-ansi|d3-color)@)'],
```
Comment (cbt1) [inline jest.config.cjs:15]: Followed the pattern provided here https://jestjs.io/docs/configuration#transformignorepatterns-arraystring for pnpm

File: jest.config.cjs [hunk 2]
```diff
@@ -5,12 +5,13 @@ module.exports = {
   testEnvironment: 'jsdom',
   testRegex: 'src/.+\\.(test|spec)\\.[jt]sx?$',
   transform: {
+    '\\.js$': 'babel-jest',
```
[the remark this answers is not in the document]
Comment (cbt1) [reply jest.config.cjs:8]: Maybe I'm missing something here. But it appears that `@swc/jest` does not transpile ESM modules to commonjs. As your suggestion does not work. That is why I ended up having to configure transform with babel-jest.

[the remark this answers is not in the document]
Comment (cbt1) [reply jest.config.cjs:8]: Oddly that is not working for me

[the remark this answers is not in the document]
Comment (cbt1) [reply jest.config.cjs:8]: Right, so deleting the lock file actually did it. Will update PR.

**Review by a-m-dev — APPROVED** [review]

File: src/utils/get-hover-color/index.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { hcl, rgb } from 'd3-color';
+
+interface ColorModifiers {
+  brighter: number;
+  darker: number;
+  opacity: number;
+}
+
+const getHoverColor = (color: string, colorModifies: ColorModifiers) => {
```
Comment (a-m-dev) [inline src/utils/get-hover-color/index.ts:9]: colorModifies -> colorModifie**rs** 
Probably auto-complete kept suggesting you this

**Review by a-m-dev — APPROVED** [review]
Just a minor comment on naming

**Review by a-m-dev — APPROVED** [review]

**Pull request merged**